src/components/MyNdla/FolderSelect.tsx

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
Flyttet en del logikk for å velge plassering i mappe-struktur hit for å gjenbruke mellom `AddResourceToFolder` og `CopyFolder`